R/customVarInfo.R

```diff
@@ -114,7 +156,7 @@
 
 #' @export
 .vsc.getAllVarInfos <- function(){
-  varInfos <- session$varInfo
+  varInfos <- session$varInfos
   varInfos <- lapply(seq(1, length(varInfos)), function(i){
```

Comment:
I suggest that all usage of `seq(1, n)` be replaced with a safer version like `seq_len(n)`.

Reply:
Thanks!